src/assets/scss/partials/_variables.scss

```diff
@@ -0,0 +1,7 @@
+@import '~@angular/material/_theming';
+
+@include mat-core();
+
+// Define colors.
+$vital-indigo: mat-color($mat-indigo, A200);
+$vital-red: mat-color($mat-red, A200);
```

Comment:
Shouldn't we try to get the primary, accent, and warn colors and create `$vital-primary`,  `$vital-accent`, and `$vital-warn`instead? So that if we change the theme, we automatically get the colors from that theme?

Reply:
![image](https://user-images.githubusercontent.com/41999054/115974605-bdf6e200-a512-11eb-93f1-fe62b19cfc02.png)
Made some changes which let us define a new theme.  For instance this is an experiment to create a dark theme.

Reply:
Wow. This looks great. The idea is very neat!